cmd/cluster-agent/commands/clusterchecks.go

```diff
@@ -51,3 +54,65 @@ func GetClusterChecksCobraCmd(flagNoColor *bool, confPath *string, loggerName co
 
 	return clusterChecksCmd
 }
+
+func RebalanceClusterChecksCobraCmd(flagNoColor *bool, confPath *string, loggerName config.LoggerName) *cobra.Command {
+	clusterChecksCmd := &cobra.Command{
+		Use:   "rebalance-clusterchecks",
+		Short: "Rebalances cluster checks",
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			if *flagNoColor {
+				color.NoColor = true
+			}
+
+			// we'll search for a config file named `datadog-cluster.yaml`
+			config.Datadog.SetConfigName("datadog-cluster")
+			err := common.SetupConfig(*confPath)
+			if err != nil {
+				return fmt.Errorf("unable to set up global cluster agent configuration: %v", err)
+			}
+
+			err = config.SetupLogger(loggerName, config.GetEnv("DD_LOG_LEVEL", "off"), "", "", false, true, false)
+			if err != nil {
+				fmt.Printf("Cannot setup logger, exiting: %v\n", err)
+				return err
+			}
+
+			return rebalanceChecks()
+		},
+	}
+
+	return clusterChecksCmd
+}
+
+func rebalanceChecks() error {
+	fmt.Println("Requesting a cluster check rebalance...")
+	c := util.GetClient(false) // FIX: get certificates right then make this true
+	urlstr := fmt.Sprintf("https://localhost:%v/api/v1/clusterchecks/rebalance", config.Datadog.GetInt("cluster_agent.cmd_port"))
+
+	// Set session token
+	err := util.SetAuthToken()
+	if err != nil {
+		return err
+	}
+
+	r, err := util.DoPost(c, urlstr, "application/json", bytes.NewBuffer([]byte{}))
+	if err != nil {
+		var errMap = make(map[string]string)
+		json.Unmarshal(r, &errMap) //nolint:errcheck
+		// If the error has been marshalled into a json object, check it and return it properly
+		if e, found := errMap["error"]; found {
+			err = fmt.Errorf(e)
+		}
+
+		fmt.Printf(`
+		Could not reach agent: %v
+		Make sure the agent is running before requesting the status.
+		Contact support if you continue having issues.`, err)
+
+		return err
+	}
+
+	fmt.Println("Cluster checks rebalanced successfully.")
```

Comment:
it would be super useful if we can print the check moves that happened

Reply:
especially if we can add a dry-run mode, so we can know the behaviour without having to do the rebalancing actually

Reply:
update: having a dry run mode requires introducing big changes in the logic without having a great value, feel free to ignore my comment 👍